grpc/src/main/java/RouterImpl.java

```diff
@@ -128,7 +128,7 @@ public void routeStreetMode(StreetRouteRequest request, StreamObserver<StreetRou
             String message = "GH internal error! Path could not be found between "
                     + ghRequest.getPoints().get(0).lat + "," + ghRequest.getPoints().get(0).lon + " to "
                     + ghRequest.getPoints().get(1).lat + "," + ghRequest.getPoints().get(1).lon;
-            logger.error(message);
+            logger.error(message, e);
```

Comment:
Does this show the stacktrace? 👍

Reply:
yup!

Reply:
We already do it for PT request errors, just forgot to add it for street requests (auto/bike/ped)